src/sparqlfetcher.ts

```diff
@@ -74,11 +73,7 @@ export class SparqlFetcher {
   }
 
   private options(body: BodyInit = ""): RequestInit {
-    const options = clone(this.fetchOptions);
-    if (body) {
-      options.body = body;
-    }
-    return options;
+    return { ...this.options, body };
```

Comment:
There's two issues here:
1. It should be `this.fetchOptions` (we are in `this.options`)
2. With this change, GET/HEAD methods will have `body: ''`. Using `node-fetch`, it throws: https://github.com/node-fetch/node-fetch/blob/bf8b4e8db350ec76dbb9236620f774fcc21b8c12/src/request.js#L77 .

    Adding the condition back would fix it:

    ```ts
      private options(body: BodyInit = ""): RequestInit {
        const options: RequestInit = { ...this.fetchOptions };
        if (body) {
          options.body = body;
        }
        return options;
      }
    ```

Reply:
D'oh 😓 Thanks for the review!